cognite/client/data_classes/assets.py

```diff
@@ -449,3 +471,292 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
         if isinstance(self.labels, LabelFilter):
             result["labels"] = self.labels.dump(camel_case)
         return result
+
+
+OnErrorOptions = Literal["ignore", "warn", "raise"]
+
+
+class AssetHierarchyValidator:
```

Comment:
I'd suggest `ValidAssetHierarchy` or just `AssetHierarchy` instead.

Reply:
I'll go with `AssetHierarchy` then, I've been back and forth quite a few times already 😅